integrated_tools.py

```diff
@@ -0,0 +1,11 @@
+def interleave(*iterable):
+    """ merge iterable objects"""
```

Comment:
Please read about the conventions of python's documentation and fix it :)
Try to describe more.